src/main/java/org/apache/sling/feature/maven/mojos/AbstractFeatureMojo.java

```diff
@@ -163,7 +163,7 @@
      * The start level for the attached jar/bundle.
      */
     @Parameter(name=FeatureProjectConfig.CFG_JAR_START_ORDER)
-    private int jarStartOrder;
```

Comment:
I think leaving this as an int is slightly better as this way Maven does the conversion and reports on the error.
You can check for greater zero later on (instead of a null check)

Reply:
ok i changed it back as you suggested: [commit](https://github.com/apache/sling-slingfeature-maven-plugin/pull/57/commits/47d9ea9c5ad3d220f6f4f361747628ec12ad2ede)